### ISSUE TITLE
[BUG] Cannot read property 'attach' of undefined

### ISSUE THREAD

**Issue by macjohnny** [bithost-gmbh/ngx-mat-select-search#146]
**Describe the bug**
With angular 8 / material 8, the following error occurs when using `<ngx-mat-select-search>` without placing it into a `<mat-option>` element:
```
preview-4adb70f742b91f09679fb.js:1 ERROR TypeError: Cannot read property 'attach' of undefined
    at MatSelectSearchComponent.setOverlayClass (mat-select-search.component.ts:444)
    at MatSelectSearchComponent.ngAfterViewInit (mat-select-search.component.ts:327)
```

**Comment by andreystorchous** [comment]
+1

**Comment by macjohnny** [comment]
this is fixed in [1.7.5](https://github.com/bithost-gmbh/ngx-mat-select-search/releases/tag/1.7.5)
